static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -82,3 +100,4 @@
 @include overide-live-blog-colours(sport, $sport-dark, $sport-pastel);
 @include overide-live-blog-colours(arts, $culture-dark, $culture-pastel);
 @include overide-live-blog-colours(lifestyle, $lifestyle-dark, $lifestyle-pastel);
+@include overide-live-blog-colours(special-report-alt, $special-report-alt-faded, $special-report-alt-dark);
```

Comment:
### Liveblog card in container without a palette

![image](https://user-images.githubusercontent.com/19683595/226293422-09bfac0f-b5a1-44aa-8901-bae2c559c242.png)